### PR TITLE
Bugfix/test db paths

### PR DESCRIPTION
**Pull request by liayoo**
🚨 Huge PR alert 🚨
- Removed the 'test' from SERVICE_TYPES, which was a temp solution for db paths used in unit & integration tests.
- Updated tests to use native app services (e.g. `/apps/<app name>/some/path`), instead of writing directly at the root (e.g. `/test/some/path`)

### CODE REVIEW

File: integration/sharding.test.js [hunk 1]
```diff
@@ -1666,13 +1665,13 @@ describe('Sharding', async () => {
         })
 
         it('ain_sendSignedTransaction with is_global = false (explicit)', () => {
-          const account = ainUtil.createAccount();
+          // const account = ainUtil.createAccount();
```
Comment (platfowner) [inline integration/sharding.test.js:1668]: remove?

File: integration/sharding.test.js [hunk 2]
```diff
@@ -1704,13 +1705,13 @@ describe('Sharding', async () => {
         })
 
         it('ain_sendSignedTransaction with is_global = true', () => {
-          const account = ainUtil.createAccount();
+          // const account = ainUtil.createAccount();
```
Comment (platfowner) [inline integration/sharding.test.js:1708]: remove?

File: integration/sharding.test.js [hunk 3]
```diff
@@ -1746,13 +1747,13 @@ describe('Sharding', async () => {
 
       describe('ain_sendSignedTransactionBatch', () => {
         it('ain_sendSignedTransactionBatch with is_global = false', () => {
-          const account = ainUtil.createAccount();
+          // const account = ainUtil.createAccount();
```
Comment (platfowner) [inline integration/sharding.test.js:1750]: ditto

File: integration/sharding.test.js [hunk 4]
```diff
@@ -1792,13 +1795,13 @@ describe('Sharding', async () => {
         })
 
         it('ain_sendSignedTransactionBatch with is_global = false (explicit)', () => {
-          const account = ainUtil.createAccount();
+          // const account = ainUtil.createAccount();
```
Comment (platfowner) [inline integration/sharding.test.js:1798]: ditto

File: integration/sharding.test.js [hunk 5]
```diff
@@ -1842,13 +1847,13 @@ describe('Sharding', async () => {
         })
 
         it('ain_sendSignedTransactionBatch with is_global = true', () => {
-          const account = ainUtil.createAccount();
+          // const account = ainUtil.createAccount();
```
Comment (platfowner) [inline integration/sharding.test.js:1850]: ditto

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks for this refactoring!

**Comment by liayoo** [comment]
Thanks for the prompt review! Now merging..

**Pull request merged**